src/components/topics-list.js

```diff
@@ -0,0 +1,274 @@
+import { html, render, nothing } from 'lit-html';
+
+export class TopicsList {
+  constructor(target, storage) {
+    this.target = target;
+    this.domTarget = undefined;
+    this.storage = storage;
+    this.storageName = 'completedTopics';
+
+    this.allTopics = [
+      {
+        name: 'People',
+        topics: [
+          {
+            title: 'Skills and knowledge',
+            form:
+              'https://api.forms.platforms.qld.gov.au/fesrqwsyzlbtegd/skillsandknowledge',
```

Comment:
how would we handle multiple environments, at least 2, dev and prd?

Reply:
This will be handled in the next sprint. It was outside the scope of our current work.